tests/CarbonInterval/ComparisonTest.php

```diff
@@ -37,4 +37,100 @@ public function testEqualToFalse()
         $this->assertFalse($oneDay->eq(CarbonInterval::hours(24)->microsecond(1)));
         $this->assertFalse($oneDay->eq(CarbonInterval::hours(23)->minutes(59)->seconds(59)->microseconds(999999)));
     }
+
```

Comment:
@burobo Add assertions for the case `self::make()` returned `null` (so `$interval = null`), for example:
```php
$this->assertFalse($oneDay->gt('xxx'));
```